airbyte-cdk/python/airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -585,6 +585,13 @@ class RemoveFields(BaseModel):
         examples=[['tags'], [['content', 'html'], ['content', 'plain_text']]],
         title='Field Paths',
     )
+    condition: Optional[str] = Field(
```

Comment:
was this file modified manually? we should run `gradle :airbyte-cdk:python:build` or `airbyte-cdk/python/bin/generate-component-manifest-files.sh` to generate it

Reply:
TIL, btw.